public/components/ctas/8.html

```diff
@@ -0,0 +1,12 @@
+<div class="bg-sky-50">
+  <div class="p-8 max-w-7xl m-auto lg:flex lg:items-center lg:justify-between sm:p-12">
+    <div>
+      <p class="text-3xl font-bold sm:text-4xl">Ready to grow? <br>
```

Comment:
The `br` is not needed, just use `block` on the `span`. I almost think we could do something like this:

```suggestion
<p>
  <strong class="text-3xl sm:text-4xl"></strong>
  <span class="block text-blue-700"></span>
</p>
```